src/index.coffee

```diff
@@ -0,0 +1,320 @@
+module.exports =
+  Client: require('./client')
+  Endpoint: require('./endpoint')
+  CreateOrder: require('./requests/create_order')
+
```

Comment:
Probably dont need 300 lines of comments? Just one github revision that shows us removing it